### PR TITLE
EZP-30064: As an editor I would like to view a content tree in location view

### PR DESCRIPTION
**Pull request by sunpietro**
| Question      | Answer
| ------------- | ---
| Tickets       | https://jira.ez.no/browse/EZP-30064
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Tests pass?   | yes/no
| Doc needed?   | yes/no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

zrzut ekranu 2019-01-29 o 16 37 17

## FOR QA AND REVIEWERS:

The feature will be implemented incrementally. That means, that the design will be changing in the forthcoming pull requests. In this PR I focused on preparing the content tree module base and implementing the basic file structure.

Requires: https://github.com/ezsystems/ezplatform-admin-ui-modules/pull/136

### CODE REVIEW

File: src/bundle/Resources/public/scss/_view.content.scss [hunk 1]
```diff
@@ -0,0 +1,14 @@
+.ez-content-view {
+    .ez-content-container {
+        display: flex;
+    }
+
+    .ez-content-tree-container {
+        min-width: 250px;
```
Comment (tischsoic) [inline src/bundle/Resources/public/scss/_view.content.scss]: Is there any reason not to use `calculateRem(250px)` here?

Comment (sunpietro) [reply src/bundle/Resources/public/scss/_view.content.scss]: Good catch.

File: src/bundle/Resources/views/layout.html.twig [hunk 2]
```diff
@@ -164,6 +164,7 @@
             '@EzPlatformAdminUiBundle/Resources/public/js/scripts/helpers/timezone.helper.js'
             '@EzPlatformAdminUiBundle/Resources/public/js/scripts/admin.notifications.js'
             '@EzPlatformAdminUiModulesBundle/Resources/public/js/UniversalDiscovery.module.js'
+            '@EzPlatformAdminUiModulesBundle/Resources/public/js/ContentTree.module.js'
```
[the remark this answers is not in the document]
Comment (sunpietro) [reply src/bundle/Resources/views/layout.html.twig]: Good catch. Fixed.

**Comment by sunpietro** [comment]
ping pong @dew326 @tischsoic

File: src/bundle/Resources/public/scss/_view.content.scss [hunk 3]
```diff
@@ -0,0 +1,14 @@
+.ez-content-view {
```
Comment (dew326) [inline src/bundle/Resources/public/scss/_view.content.scss]: The files should be named `_content-view.scss`. Here the pattern is with `-`.

File: src/bundle/Resources/views/content/locationview.html.twig [hunk 4]
```diff
@@ -170,6 +174,7 @@
         '@EzPlatformAdminUiBundle/Resources/public/js/scripts/button.state.toggle.js'
         '@EzPlatformAdminUiBundle/Resources/public/js/scripts/admin.version.edit.conflict.js'
         '@EzPlatformAdminUiBundle/Resources/public/js/scripts/admin.location.bookmark.js'
+        '@EzPlatformAdminUiModulesBundle/Resources/public/js/ContentTree.module.js'
```
Comment (dew326) [inline src/bundle/Resources/views/content/locationview.html.twig]: Is it working? The JS which needs this file is added before. This should be moved to the `react_modules` block.

File: src/bundle/Resources/public/scss/_view.content.scss (hunk 3 above)
Comment (sunpietro) [reply src/bundle/Resources/public/scss/_view.content.scss]: I disagree. In this repo we have a convention with dots.

**Comment by sunpietro** [comment]
ping @dew326

**Pull request merged**